README.md

```diff
@@ -1,94 +1,83 @@
 # RIA2_rekognition_python
 
-# Get Started
+## Get Started
 
-First install the dependencies
+### Setup environment variables
 
-```sh
-pip install -r requirements.txt
-```
-## Configure the `.env` file
-
-The `.env` file is a file fill up with the secure configurations to link your project with AWS.
-
-1. Copy Past the file `.env_example` and rename it to `.env`
+1. Copy paste the file `.env_example` and rename it to `.env`
 
-2. Configure it
+2. Fill it with your settings
 
 ```js
-BUCKET_NAME=bucket
-BUCKET_URL=s3://${BUCKET_NAME}.${DOMAIN}
-BUCKET_FOLDER=default_storage
+BUCKET_NAME=bucket // Your aws bucket name
 AWS_ACCESS_KEY_ID=aws_access_key_id // Your aws access key id
 AWS_SECRET_ACCESS_KEY=aws_secret_access_key // Your aws secret access key
-AWS_DEFAULT_REGION=eu-central-1
-STORAGE_FOLDER=C:/ //The storage folder where images will be downloaded
+AWS_DEFAULT_REGION=eu-central-1 // Your aws default region
+STORAGE_FOLDER=C:/ // Storage folder where images will be downloaded
```

Comment:
Potential compatibility issue with other system than windows

Reply:
Yes probably, do you have any solutions since you are the only one working on linux?